src/extensions/mountainview/MVSortingView/MVSortingView.tsx

```diff
@@ -134,18 +136,20 @@ const MVSortingView: FunctionComponent<SortingViewProps> = (props) => {
     }, [openViewsDispatch])
     const width = props.width || 600
     const height = props.height || 900
-    const filterIcon = <span style={{color: 'gray'}}><FontAwesomeIcon icon={faSocks} type="outline" /></span>
+    const preprocessingIcon = <span style={{color: 'gray'}}><FontAwesomeIcon icon={faSocks} type="outline" /></span>
```

Comment:
These icons are placeholders for now - will need to choose more representative icons.

Reply:
I'd been wondering about the socks...